packages/sns/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@message-queue-toolkit/sns",
-  "version": "12.1.1",
+  "version": "13.0.0",
```

Comment:
before we release this version, let's also merge https://github.com/kibertoad/message-queue-toolkit/pull/121, so that we don't need to do semver majors in a row

Reply:
Sounds great to me 🙇 I will wait to merge this PR

Reply:
feel free to merge, just don't hurry releasing it :D.
After you are done, I will address any conflicts if there are on the other PR, and release both changes

Reply:
Ah, right, this repo doesn't have auto-release 😅 sorry, I was confused, thanks for the clarification 🙏